fuse_examples/classification/duke_breast_cancer/dataset.py

```diff
@@ -161,7 +161,7 @@ def duke_breast_cancer_dataset(paths,train_common_params,lgr):
                                          )
 
     lgr.info(f'- Load and cache data:')
-    train_dataset.create()
+    train_dataset.create(num_workers=0)
```

Comment:
You shouldn't set the num_workers here (in train set) to 0. It should work as-is. Can you please change it back?